src/main/java/rx/plugins/RxJavaPlugins.java

```diff
@@ -141,7 +156,7 @@ public void registerObservableExecutionHook(RxJavaObservableExecutionHook impl)
         }
     }
 
-    private static Object getPluginImplementationViaProperty(Class<?> pluginClass) {
+    /* test */ static Object getPluginImplementationViaProperty(Class<?> pluginClass, Properties props) {
```

Comment:
I meant not annotation but comment.

On Wed, Nov 11, 2015, 21:32 Shixiong Zhu <notifications@github.com> wrote:

> In src/main/java/rx/plugins/RxJavaPlugins.java
> <https://github.com/ReactiveX/RxJava/pull/3513#discussion_r44567271>:
>
> > @@ -141,7 +156,7 @@ public void registerObservableExecutionHook(RxJavaObservableExecutionHook impl)
> >          }
> >      }
> >
> > -    private static Object getPluginImplementationViaProperty(Class<?> pluginClass) {
> > +    /* test */ static Object getPluginImplementationViaProperty(Class<?> pluginClass, Properties props) {
>
> VisibleForTesting is in Guava
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ReactiveX/RxJava/pull/3513/files#r44567271>.
>
-- 

@artem_zin